### ISSUE TITLE
Facing error while trying to extract package

### ISSUE THREAD

**Issue by nikhilnfer** [elastic/go-elasticsearch#153]
```
go: extracting github.com/elastic/go-elasticsearch/v7 v7.7.0
# github.com/elastic/go-elasticsearch/v7/estransport
../../../../go/pkg/mod/github.com/elastic/go-elasticsearch/v7@v7.7.0/estransport/estransport.go:121:32: httpTransport.Clone undefined (type *http.Transport has no field or method Clone)
```

**Comment by karmi** [comment]
Seems like you're using an older version of Go, the transport.clone is available since 1.13, see https://github.com/elastic/go-elasticsearch/issues/134

**Comment by nikhilnfer** [comment]
That seems to be the issue .It was failing on 12.9 , but working on 13.3 . Thanks for replying.
